scripts/create_py_tags.py

```diff
@@ -29,13 +28,12 @@
 import os
 import re
 import sys
-import types
 
 PYTHON_LIB_DIRECTORY = os.path.dirname(os.__file__)
-PYTHON_LIB_IGNORE_PACKAGES = (u'test', u'dist-packages', u'site-packages', 'Tools')
+PYTHON_LIB_IGNORE_PACKAGES = ('test', 'dist-packages', 'site-packages', 'Tools')
 # some modules execute funky code when they are imported which we really don't want here
 # (though if you feel funny, try: 'import antigravity')
-PYTHON_LIB_IGNORE_MODULES = (u'antigravity.py', u'idlelib/idle.py', u'multiprocessing/util.py')
+PYTHON_LIB_IGNORE_MODULES = ('antigravity.py', 'idlelib/idle.py', 'multiprocessing/util.py')
```

Comment:
Additionally, we should exclude `idlelib/__main__.py` as otherwise the Idle IDE is started.
And we could even exclude `this.py` as it contains only the Zen of Python but no tags. So we save us printing the Zen of Python.